### PR TITLE
React: Add Index Page

### PR DESCRIPTION
**Pull request by rabeehrz**
Add Index Page

Things that are not added:

- The Hero Section: I have added it and it works fine on my computer, But it doesn't work on deployment. I have no idea why. The images are not loading properly. It would be great if someone could take a look at that. I have commented it out.

- The confetti for achievements: It is going to change in master, So it would be best if it was implemented after the changes in master.

https://girlscript-chennai-website-deploy.now.sh/

### CODE REVIEW

File: src/styles/global.css [hunk 1]
```diff
@@ -255,15 +254,11 @@ svg {
...
-.team-part {
+<<<<<<< HEAD ======= .team-part {
   color: var(--theme-color);
 }
 
-
-
-
-.wave {
+>>>>>>>3edbfd502a5989e65f242481b6d9a1f2e246735b .wave {
```
Comment (aashimgarg) [inline src/styles/global.css:261]: you can remove this

Comment (aashimgarg) [reply src/styles/global.css:261]: by the way global.css was refactored to global.scss

Comment (rabeehrz) [reply src/styles/global.css:261]: Thank You! I didn’t notice that. I’ll make the changes later and update the PR.

Comment (rabeehrz) [reply src/styles/global.css]: > by the way global.css was refactored to global.scss

I can't find it on the latest react_version commit. Did someone remove it?

**Pull request merged**